.github/CODEOWNERS

```diff
@@ -8,6 +8,7 @@
 
 # Common Files
 Datadog.Trace.Trimming.xml                @DataDog/apm-dotnet
+Directory.Build.props                     @DataDog/apm-dotnet
```

Comment:
We should move this towards the bottom of the file so it is not overridden below.